libr/anal/fcn.c

```diff
@@ -2132,7 +2132,7 @@ R_API RAnalBlock *r_anal_fcn_bbget(RAnalFunction *fcn, ut64 addr) {
 	RListIter *iter;
 	RAnalBlock *bb;
 	r_list_foreach (fcn->bbs, iter, bb) {
-		if (bb->addr == addr) {
+		if (addr >= bb->addr && addr < (bb->addr + bb->size)) {
```

Comment:
this change is risky. This function was intended to return a RAnalBlock exactly at the given address.
For example, in disasm.c it is used to know whether an address was the start of a basicblock.

I suggest we create a separate function `r_anal_fcn_bbget_in`, like what we do for function where we have `_at` and `_in`.

Reply:
yeah we could use a map to improve the performance, but until it's not a real problem i guess it's fine